packages/page-accounts/src/modals/Create.tsx

```diff
@@ -61,10 +61,17 @@ interface DeriveValidationOutput {
 const DEFAULT_PAIR_TYPE = 'sr25519';
 const STEPS_COUNT = 3;
 
+function isHDDerivePath (path: string) {
```

Comment:
For validation, use `hdValidatePath` from `util-crypto`. (It is slightly more strict, no uppercase "M" and it needs to not be naked, i.e. `m/` is not valid, `m/0` would be. Also checks for valid numeric derivations and sizes)

Reply:
Module '"../../../../node_modules/@polkadot/util-crypto"' has no exported member 'hdValidatePath'.ts(2305)

Reply:
oh should I just use ledgerValidatePath?

Reply:
Merge master.